docs/docs/mdx/programmatically-creating-pages.md

```diff
@@ -224,19 +224,27 @@ will be rendered as the template for all posts. There's a component,
 `MDXRenderer` which is used by `gatsby-plugin-mdx` that will be used to render any
 programmatically accessed MDX content.
 
+For now, to update imports within .mdx files, you should re-run your gatsby development environment. Otherwise, it will raise a ReferenceError. To import things dynamically, you can use the `MDXProvider` component and provide it all the common components you'll be using, such as `Link`.
```

Comment:
```suggestion
For now, to update imports within .mdx files, you should rerun your Gatsby development environment. Otherwise, it will raise a `ReferenceError`. To import things dynamically, you can use the `MDXProvider` component and provide it all the common components you'll be using, such as `Link`.
```